src/docs/03-Code Guidelines/05-Browser-Support.md

```diff
@@ -1,12 +1,32 @@
 All contents in YouSee DNA is build for and tested in the browsers below. Any YouSee site or web application must be tested in the same browsers.
 
 ## Desktop
-- Internet Explorer 11
-- Chrome (2 latest versions) for Windows and MacOS
-- Firefox (2 latest versions) for Windows and MacOS
-- Safari (2 latest versions) for MacOS
-- Edge (2 latest versions) for Windows
-
-## Mobile/Tablet
-- Chrome (2 latest versions) for iOS and Android
-- Safari (2 latest versions) for iOS
+- Chrome (3 latest versions) for Windows, MacOS and Ubuntu <sup>[[1]](#ref1)</sup>
+- Firefox (3 latest versions) for Windows, MacOS and Ubuntu <sup>[[1]](#ref1)</sup>
+- Safari (2 latest versions) for MacOS <sup>[[2]](#ref2)</sup>
+- Edge (3 latest versions) for Windows <sup>[[1]](#ref1)</sup>
+
+
+## Phone & Tablet
+- Chrome (3 latest versions) for Android and iOS <sup>[[1]](#ref1)</sup>
+- Safari (2 latest versions) for iOS <sup>[[2]](#ref2)</sup>
+- Samsung Internet (latest version) for Android <sup>[[3]](#ref3)</sup>
+
+
+## Partial support
+- Internet Explorer 11 for Windows <sup>[[4]](#ref4)</sup>
+
+
+## No support
+- Internet Explorer 10 and older
+- Opera and Opera Mini
+- Console and Television based browsers
+
+
+## Notes
+Latest version cover only the major release version, patches and minor updates is not taken into consideration.
+
+1. <a name="ref1"></a>Chrome, Firefox and Edge has a short regular release cycle, support for 2 prior version is necessarily.

```

Comment:
This is invalid HTML — instead write the ordered list completely, like this:
```
<ol>
  <li id="ref1">....</li>
  <li id="ref2">....</li>
  etc
</ol>
```

Reply:
Also, the last part of the sentence should be:
"support for 2 prior versions is necessary."